docs/src/manifolds/stiefel.md

```diff
@@ -44,4 +44,24 @@ Pages = ["manifolds/StiefelCanonicalMetric.jl"]
 Order = [:function]
 ```
 
+# The submersion or normal metric
+
+```@autodocs
+Modules = [Manifolds]
+Pages = ["manifolds/StiefelSubmersionMetric.jl"]
+Order = [:type, :function]
+Public = true
+Private = false
+```
+
+# Internal types
+
+```@autodocs
+Modules = [Manifolds]
+Pages = ["manifolds/StiefelSubmersionMetric.jl"]
+Order = [:type, :function]
```

Comment:
Maybe just as a small remark – the section is currently titled to cover internal types but the code might also display internal functions. We could unify that by removing functions here or adding functions in the title.